osu.Desktop/Windows/WindowsAssociationManager.cs

```diff
@@ -40,8 +40,8 @@ public static class WindowsAssociationManager
 
         private static readonly FileAssociation[] file_associations =
         {
-            new FileAssociation(@".osz", WindowsAssociationManagerStrings.OsuBeatmap, Icons.Lazer),
-            new FileAssociation(@".olz", WindowsAssociationManagerStrings.OsuBeatmap, Icons.Lazer),
+            new FileAssociation(@".osz", WindowsAssociationManagerStrings.OsuBeatmap, Icons.Beatmap),
+            new FileAssociation(@".olz", WindowsAssociationManagerStrings.OsuBeatmap, Icons.Beatmap),
             new FileAssociation(@".osr", WindowsAssociationManagerStrings.OsuReplay, Icons.Lazer),
             new FileAssociation(@".osk", WindowsAssociationManagerStrings.OsuSkin, Icons.Lazer),
```

Comment:
Since we're here redesigning icons, how about these?

FWIW stable also just uses the music icon for these, so we don't have a reference icon to go by if we want distinct icons.

Reply:
@arflyte